xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/ApplicationResources.properties

```diff
@@ -29,10 +29,25 @@ mentions.event.mention.description=mentioned you on page
 mentions.event.mention.description.document=mentioned you on page
 mentions.event.mention.description.comment=mentioned you on a comment on page
 mentions.event.mention.description.annotation=mentioned you on an annotation on page
-mentions.event.mention.description.awmField=mentioned you on a field on the AWM document
+mentions.event.mention.description.textField=mentioned you on a field on document
```

Comment:
I think there are parameters missing in these translations, you cannot assume that a sentence structure "[User] ... [Document]" is possible in all languages. While this may be hard to change for existing translations, I think at least no new keys with this problem should be introduced.

Reply:
The `[Document]` would be easy to fix. For the `[User]` it needs to be checked since for now [`displayEventDetail`](https://github.com/manuelleduc/xwiki-platform/blob/0dd5eec56bceeee20b0366b92a1c89f094ea32d2/xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/mentions/mention.vm#L36) is taking care of inserting the User link.

Reply:
yes right now it's the standard way of displaying notifications with the existing macros, I don't think we should fix this here as this targets LTS. We'd need to rework all those macros with new ones probably.